packages/mobile/src/components/DevSkipButton.tsx

```diff
@@ -18,7 +18,11 @@ interface DispatchProps {
   devModeTriggerClicked: typeof devModeTriggerClicked
 }
 
-type Props = { nextScreen: keyof StackParamList; onSkip?: () => void } & StateProps & DispatchProps
+type Props = {
+  nextScreen?: keyof StackParamList
+  onSkip?: () => void
```

Comment:
nit: would be nice to separate the 2 use cases and enforce them with TypeScript.
if `nextScreen` is not specified, `onSkip` should be set.

```tsx
type Props = {
  nextScreen: keyof StackParamList
  onSkip?: () => void
} | {  onSkip: () => void }
```